core/iwasm/common/wasm_application.c

```diff
@@ -18,6 +18,9 @@
 #if WASM_ENABLE_STRINGREF != 0
 #include "string_object.h"
 #endif
+#if WASM_ENABLE_GC_PERF_PROFILING
```

Comment:
Had better use `WASM_ENABLE_GC_PERF_PROFILING != 0`, we usually use that coding style.